a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -219,122 +219,187 @@ <h3>Check for Node</h3>
+                                <li>otherwise if false it means that this metadata is not present.</li>ff
+                            </ul>
+                            <p>This means that there are textual alternatives for images.</p>
+                        </dd>
+                    </dl>
+                    <h5>Variables setup</h5>
+                    <ol class="condition">
+                        <li><b>LET</b> <var>package_document</var> be the result of calling <a href="#pre-processing">preprocessing</a> given <var>package_document_as_text</var>.</li>
 
-					<li>
-						<span><b>IF</b> <var>all_necessary_content_textual</var>:</span>
-						<span><b>THEN</b> display <code id="nonvisual-reading-readable">"Readable in read aloud or dynamic braille"</code>.</span>
-					</li>
-					<li>
-						<span><b>ELSE IF</b> <var>non_textual_content_images</var> <b>AND</b> <b>NOT</b> <var>textual_alternative_images</var>:</span>
-						<span><b>THEN</b> display <code id="nonvisual-reading-not-fully">"Not fully readable in read aloud or dynamic braille"</code>.</span>
-					</li>
-					<li><b>ELSE</b> display <code id="nonvisual-reading-may-not-fully">"May not be fully readable in read aloud or dynamic braille"</code>.</li>
-				</ol>
+                        <li><b>LET</b> <var>all_necessary_content_textual</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessModeSufficient</i>" and normalize-space()="<i>textual</i>"]</code>.</li>
 
+                        <li>
+                            <b>LET</b> <var>non_textual_content_images</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="<i>schema:accessMode</i>" and normalize-space() = ("chartOnVisual", "chemOnVisual", "diagramOnVisual", "mathOnVisual", "musicOnVisual", "textOnVisual")]</code>.
+                        </li>
 
-			</section>
+                        <li>
+                            <b>LET</b> <var>textual_alternative_images</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="<i>schema:accessibilityFeature</i>" and normalize-space()=("<i>longDescription</i>", "<i>alternativeText</i>","<i>describedMath"</i>)]</code>.
+                        </li>
+                    </ol>
+                    <h5>Instructions</h5>
+                    <ol class="condition">
+                                <li>
+                                    <span><b>IF</b> <var>textual_alternative_images</var>:</span>
+                                    <span><b>THEN</b> display <code id="ways-of-reading-nonvisual-reading-alt-text">"Has alt text"</code>.</span>
```

Comment:
Does outputting that there is alt text still make sense as a distinct string under "ways of reading"? The context of these two statements belonging to nonvisual reading is kind of lost now.